### ISSUE TITLE
plotting methods other than PCA in compare app

### ISSUE THREAD

**Issue by lmsimp** [ComputationalProteomicsUnit/pRolocGUI#94]
The compare app doesn't allow you to specify a plotting method. We set the plotting method internally to be PCA. I would like to change the code so we can allow other visualisations. This of course is tricky if we want to use the remap function as this only works with PCA. We could allow a call to `method` e.g. `method = "t-SNE"` only if `remap = FASLE` then this wouldn't break the call to `remap`. What do you think @lgatto

**Comment by lgatto** [comment]
The question is (and I think we have different opinions here) whether it makes sense to compare datasets that haven't been remapped. If we believe that some sort of remapping/normalisation is essential, then we would need to provide one before offering the visualisation. Let's discuss next week.

NB: there is no remap-like function for t-SNE (at least not in R, and I'm not sure there is one at all).

**Comment by lmsimp** [comment]
Yes, re-map is straight forward with PCA and I think it should be the default.  

I think it is useful however, to allow one to load two different datasets and compare locations. Does this make sense if there is no-remapping? I think the answer depends on the question and that is for the user to decide. My current use case is allowing me to look at the localisations assigned from `phenoDisco` and the predicted locations from curated markers. The compare app is a nice way to do this, however, I do not want to re-map and I would like to visualise with t-SNE.
